ultravox/training/train.py

```diff
@@ -211,7 +226,7 @@ def main() -> None:
     trainer = transformers.Seq2SeqTrainer(
         model,
         train_dataset=train_dataset,
-        eval_dataset=val_dataset,
+        eval_dataset=val_datasets,
```

Comment:
does this work OK as a collection, or do we need to wrap this dict of datasets with a Combine or Interleave wrapper?

Reply:
Yes it works. If you give it a dict, then all of them are run separately and you get their metrics in W&B separately as well.

That's why if you look at the image at the top of the PR, you can see curves for: "peoplespeech", "text_peoplespeech", "soda", "text_soda", and so on.